pandas/core/frame.py

```diff
@@ -3287,7 +3306,9 @@ def _set_item(self, key, value):
 
         self._set_item_mgr(key, value)
 
-    def _set_value(self, index, col, value, takeable: bool = False):
+    def _set_value(
+        self, index: int, col, value: Scalar, takeable: bool = False
```

Comment:
the docstring states `index : row label`, is `index: int` correct here?

maybe need to update the docstring to the
```
parameter: type
    description
```
format

Reply:
right, so should be `Label` I think (again, my bad)

Docstring updated.